### PR TITLE
feat - adding part 1

### PR DESCRIPTION
**Pull request by hmoumani**

### CODE REVIEW

File: P1/P1.gns3project
Comment (abel-haj) [inline P1/P1.gns3project:1]: Chnahwa hadchi

File: P1/README.md
Comment (abel-haj) [inline P1/README.md:1]: Hadchi hzitih mlintra? (privacy policy)

File: P1/_hmoumani-1_host
Comment (abel-haj) [inline P1/_hmoumani-1_host:1]: Hadchi fkhbar dawla (allowed?)

File: P1/P1.gns3project
Comment (hmoumani) [reply P1/P1.gns3project:1]: binary file, to be imported to GNS3, it's basically where we import the two configured docker images

File: P1/README.md
Comment (hmoumani) [reply P1/README.md:1]: generated

File: P1/_hmoumani-1_host
Comment (hmoumani) [reply P1/_hmoumani-1_host:1]: what are you taking about?

**Comment by abel-haj** [comment]
Good to go! Well done!

**Pull request merged**